nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "watch": ["src"],
+    "ext": "ts,json,env",
+    "ignore": ["src/**/*.spec.ts"],
+    "exec": "ts-node-dev --transpile-only ./src/index.ts"
+  }
```

Comment:
Add an extra line. For future reference, always add an extra line at the end of every file. It can be seen by that 'red minus sign'

Reply:
got it